tests/no_exit_message.rs

```diff
@@ -129,3 +129,67 @@ hello:
   stderr: "",
   status: 100,
 }
+
+test! {
+  name: recipe_exit_message_setting_suppressed,
+  justfile: r#"
+set no-exit-message
+
+# This is a doc comment
+hello:
+  @echo "Hello, World!"
+  @exit 100
+"#,
+  stdout:   "Hello, World!\n",
+  stderr:   "",
+  status:   100,
+}
+
+
+test! {
+  name: shebang_exit_message_setting_suppressed,
+  justfile: r"
+set no-exit-message
+
+hello:
+  #!/usr/bin/env bash
+  echo 'Hello, World!'
+  exit 100
+",
+  stdout: "Hello, World!\n",
+  stderr: "",
+  status: 100,
+}
+
+test! {
+  name: exit_message_attribute_and_setting,
+  justfile: r"
+set no-exit-message
+
+[no-exit-message]
+hello:
+  echo 'Hello, World!'
+  exit 100
+",
+  stdout: "Hello, World!\n",
+  stderr: r#"
+    echo 'Hello, World!'
+    exit 100
+  "#,
+  status: 100,
+}
+
+
+test! {
```

Comment:
I think this test can be removed, since whether or not a recipe is silent doesn't interact with exit messages.